stack.yaml

```diff
@@ -28,6 +28,11 @@ resolver: lts-19.6
 #   subdirs:
 #   - auto-update
 #   - wai
+
+# FIXME: A lot of dependency contraints are not satisfied when building, after
+# bumping the resolver, but adding this allows for building.
+allow-newer: true
+
```

Comment:
We should discuss this.

Reply:
We don't want that. If it is not possible to update without it we will postpone the update.

Reply:
Out of curiosity, what dependencies are causing the problem?

Reply:
It was seemingly caused by the dependency on `Concordium/http2-grpc-haskell` which also had `allow-newer: true`. I removed it, bumped contraints and fixed the resulting warnings in that, and that seemed to resolve it here by transitivity.